src/general_manager/interface/databaseBasedInterface.py

```
@@ -0,0 +1,537 @@
+                    self._instance, field_name
+                )
+
+        for field_name in cls.__getForeignKeyFields():
+            related_model = cls._model._meta.get_field(field_name).related_model
+            if related_model and hasattr(
+                related_model,
+                "_general_manager_class",
+            ):
+                generalManagerClass = related_model._general_manager_class
+                field_values[f"{field_name}"] = (
+                    lambda self, field_name=field_name: generalManagerClass(
```

Ruff (0.11.9)

Function definition does not bind loop variable `generalManagerClass`

(B023)